javascript/javascript2/week2/hyf-bay/main.js

```diff
@@ -0,0 +1,67 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+const list = document.querySelector("ul");
+let showedProduts = products;
+
+function renderProducts(productArray) {
+    // your code here
```

Comment:
Great job, but I suggest to create those elements in `html` file where it will be easier to style them.  Here you can use `querySelector` instead of `createElement`